cmd/operator/internal/controller/kubearchiveconfig_controller.go

```diff
@@ -96,37 +107,43 @@ func (r *KubeArchiveConfigReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{}, nil
 	}
 
-	_, err := r.reconcileA14eServiceAccount(ctx, kaconfig)
+	cm, err := r.reconcileFilterConfigMap(ctx, kaconfig)
```

Comment:
`kaconfig` is not used.

```suggestion
	cm, err := r.reconcileFilterConfigMap(ctx)
```

Reply:
Fixed.